Class/practice2.1/taskTask3.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Compile the C++ program
+cd ~/ProfProg/Class/practice2.1 || exit
+clang++ -std=c++23 -Wall -Wextra -Wpedantic -Werror task3.cpp -o task3
+
+# Run Test Cases
+echo "Running Test Cases:"
+
+# Test Case 1: First Run with a New Name
```

Comment:
To make the real isolated test cases we should have some prerequisite that will reset environment predictably - in this case database file should be either deleted or set to some value.